central/clusterinit/service/service_impl.go

```diff
@@ -9,16 +9,46 @@ import (
 	"github.com/stackrox/rox/central/clusterinit/backend"
 	"github.com/stackrox/rox/central/clusterinit/store"
 	v1 "github.com/stackrox/rox/generated/api/v1"
-	"github.com/stackrox/rox/pkg/defaults/accesscontrol"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/grpc/authz"
+	"github.com/stackrox/rox/pkg/grpc/authz/perrpc"
 	"github.com/stackrox/rox/pkg/grpc/authz/user"
+	"github.com/stackrox/rox/pkg/sac/resources"
 	"github.com/stackrox/rox/pkg/set"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 var (
-	authorizer = user.WithRole(accesscontrol.Admin)
+	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
+		user.With(
+			permissions.View(resources.Administration),
+			permissions.View(resources.Integration),
+		): {
+			"/v1.ClusterInitService/GetCAConfig",
+		},
+		user.With(
+			permissions.View(resources.Administration),
+			permissions.View(resources.Cluster),
+			permissions.View(resources.Integration),
```

Comment:
Is this an `OR` which would mean that a user with one of the roles can access the init-bundles?

Reply:
Some things I do not understand and need clarification, I miss some context on SAC and roles in ACS so questions might be obvious:

- Why is administrator not sufficient alone to see the resources? What is a above an administrator?
- Why is it okay for a user without `resources.Cluster` to revoke but not to generate an init-bundle? 

**Concern**
I am worried that a user locks themselves out of init-bundles by accidentially removing a permission, the user would need to re-construct the permission set for init-bundles.

Reply:
@SimonBaeumer Quick answer why this investigation is current behavior limits authorization to only system **Admin role** and not any user-created role that has appropriate permissions. Conditional logic in UI to omit unauthorized parts (for example, **Cluster init bundles** card on Integrations page) is based on **permissions of roles** (that is, access levels for resources) not on a **specific role**.

Reply:
With these SAC changes, what does change for the customer? Does the customer needs advice?

Reply:
Yes, that is an important question. A thread in forum-acs-authn-authz is discussing it.
* On one side of the coin, customer has opportunity to **reduce** the amount of permissions for a role that they create for people who create secured clusters.
* On other side of the coin, it seems like customer needs to review existing permission sets to verify that any roles which have whatever become the minimal permission and therefore **would have** authorization to create secured clusters, **should have** authorization. Unless the ability to create via init bundles is more risky than to create via 2 downloads on clusters page, then it seems like the upside outweighs the downside.

Reply:
Okay, I can't give an approval alone on this, I do not feel comfortable with the context I have on SAC. 
I think it is good that I am on review too, but a secondary reviewer with more knowledge about SAC is necessary here @rhybrillou.

Reply:
Sure @SimonBaeumer. Initially, this was a proof of concept on reducing the required permissions from the built-in `Admin` role to the minimum required permission set.
The merlin team is also tagged for review. I'll check internally who has enough context on access control to review this change.